website/docs/api/admin/feature-toggles-api.md

```diff
@@ -253,7 +301,11 @@ None
 }
 ```
 
-### Disable a Feature Toggle {#disable-a-feature-toggle}
+## Disable a Feature Toggle {#disable-a-feature-toggle}
+
+:::caution Deprecation notice
+This endpoint is deprecated. Please use the [project-based endpoint to disable feature toggles](./feature-toggles-api-v2.md#where?) instead.
```

Comment:
How do you disable feature toggles in v2 of the API?

Reply:
One does not disable a feature toggle on top level, instead you disable on a per-environment basis. Looks like we have some documentation debt to update/fix

Disabling/Enabling an environment is done via POST calls to
https://app.unleash-hosted.com/hosted/api/admin/projects/:projectId/features/:toggleName/environments/:envName/off for disabling

and
https://app.unleash-hosted.com/hosted/api/admin/projects/:projectId/features/:toggleName/environments/:envName/on
for enabling.

For enabling, the environment must at least have one strategy